network/src/commonMain/kotlin/com/wire/kalium/network/api/featureConfigs/FeatureConfigApi.kt

```diff
@@ -0,0 +1,24 @@
+package com.wire.kalium.network.api.featureConfigs
+
+import com.wire.kalium.network.AuthenticatedNetworkClient
+import com.wire.kalium.network.utils.NetworkResponse
+import com.wire.kalium.network.utils.wrapKaliumResponse
+import io.ktor.client.request.get
+
+
```

Comment:
```suggestion

```

Reply:
we can add this rule to detekt , so we can't have multiple lines 
I will mention it in the collective